src/utils/npm/npm-prefix.js

```diff
@@ -0,0 +1,16 @@
+const { prefix } = require("../prefix");
```

Comment:
Could we combine the `prefix.js` and `npm-prefix.js` files?

Reply:
Sure!

Reply:
Hmm... tried doing this and it becomes much harder to write tests because of this:
https://github.com/facebook/jest/issues/936#issuecomment-542410970

Reply:
I can push my branch and maybe you can figure out how to make this work, or I can leave the files where they are. Let me know your preference.

Reply:
Okay, I'll take a look once you've pushed :)